plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/runner/RequirementsRunner.java

```diff
@@ -301,6 +302,28 @@ protected boolean isIgnored(FrameworkMethod child) {
 	public void setRequirementsInjector(RequirementsInjector requirementsInjector) {
 		this.requirementsInjector = requirementsInjector;
 	}
+	
+	/**
+	 * Returns true if operation is AND and all condition classes return true in shouldRun(child).
+	 * Returns true if operation is OR and at least one of condition classes returns true in shouldRun(child).
+	 * Otherwise, returns false.
+	 * 
+	 * @param runIfAnnotation
+	 * @param child
+	 * @return
```

Comment:
add description to params and what is returned, if javadoc is provided for private methods, it should still be correct